src/org/aigents/nlp/lg/Disjunct.java

```diff
@@ -0,0 +1,21 @@
+package org.aigents.nlp.lg;
+
+import java.util.*;
+import java.io.*;
+
+public class Disjunct {
+	String disjunct, classname;
```

Comment:
I think you need to have more deep structure, listing all connectors of a disjunct in given order, having every connector be either left (-) or right (+). 

I am not sure that you need a classname here, please make sure why do you need that.